ai/GameDomainObjects/ball.py

```diff
@@ -6,9 +6,9 @@
 
 
 class Ball:
-    def __init__(self, id):
+    def __init__(self, id, position=Position()):
```

Comment:
id shadow builtin function. since the ai doesnt care with multiple ball, it should be remove. The game_state sent by the engine will alway have no more than 1 ball.